src/middleware/error.js

```diff
@@ -31,21 +31,11 @@ function errorHandler (err, req, res, next) {
   })
 }
 
-/**
- * Used as a 404 default for routes
- */
-function notFound (req) {
-  // TODO: when there's a 404 page to be rendered here,
-  // replace the json return with html
-  throw new NotFoundError(`Page not found! ${req.method} ${req.originalUrl}`)
-}
-
 function methodNotAllowed (req) {
```

Comment:
For logging, what if you kept the `notFound` function, and instead of throwing an error, just log the error in there and then renders the page?

Reply:
Yeah that works for me. I'm not entirely sure about the format of our log messages though. What do you think about 500e23c5?

Reply:
Works for me